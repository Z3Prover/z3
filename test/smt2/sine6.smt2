
; Copyright (c) 2015 Microsoft Corporation

(set-option :numeral-as-real true)
(declare-const x Real)
(declare-const y Real)
(declare-const a Int)

(simplify (sin (+ x (* 2 pi) y (* 2 pi (to_real a)))))
(simplify (sin (+ x (* 2 pi) y (* 4 pi (to_real a)))))
(simplify (sin (+ x (* 2 pi) y (* (- 4) pi (to_real a)))))
(simplify (sin (+ x (* 2 pi) y (* (- 2) pi (to_real a)))))
(simplify (sin (+ x (* 2 pi) y (* pi (to_real a)))))

(simplify (sin (+ x (* 2 pi) y (* 2 (to_real a) pi))))
(simplify (sin (+ x (* 2 pi) y (* 4 (to_real a) pi))))

