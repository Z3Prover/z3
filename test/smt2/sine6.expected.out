(sin (+ x y))
(sin (+ x y))
(sin (+ x y))
(sin (+ x y))
(sin (+ x y (* pi (to_real a))))
(sin (+ x y))
(sin (+ x y))
